Admin/Admin.php

```diff
@@ -784,9 +777,9 @@ protected function buildList()
 
         if (count($this->getBatchActions()) > 0) {
             $fieldDescription = $this->getModelManager()->getNewFieldDescriptionInstance($this->getClass(), 'batch', array(
-                'label'         => 'batch',
```

Comment:
I was sure you will stuck on this! :-)

Well, I'll run with `align_double_arrow` and `align_equals` contrib fixers.